cfp.html

```diff
@@ -36,10 +40,9 @@ <h1 style="margin-top: 50px" class="section-header align-self-start">Speaker ass
             <p>If you’re a first time speaker and thinking EuRuKo is out of your reach, you’re mistaken! We encourage first-time speakers to use EuRuKo as a springboard for a career of awesome talks!</p>
             <p>Ruby’s in the name, so Ruby should be the game! Rails talks are cool, of course, but the content of the conference is framework-agnostic. And hey, if you’ve got a non-Ruby or ‘for Rubyists’ talk in mind, send it our way!</p>
             <h1 style="margin-top: 50px" class="section-header align-self-start">The perks</h1>
-            <p>What do you get as a speaker, I hear you 🤔’ing? If you’re selected as a speaker, here’s what we can offer you:</p>
+            <p>What do you get as a speaker, I hear you asking? Aside from having free entry to the conference (which really barely counts as a perk!), if you’re selected as a speaker, here’s what we can offer you:</p>
             <ul>
-                <li>A free ticket to the conference</li>
-                <li>If your employer can cover your travel and accommodation, that’d be amazing! You could even show them on a slide!</li>
+                <li>We will cover travel and accommodation for you. If your employer can cover your travel and accommodation though, that’d be amazing! You could even show them on a slide!</li>
                 <li>Please let us know how we can help you out otherwise!</li>
             </ul>
             <p>For details and the submission procedure, please take a look at our CFP app:</p>
```

Comment:
Maybe add "or for any questions regarding visiting the conference as a speaker, email us at ..."